src/bio/README.md

```diff
@@ -179,12 +179,10 @@ Pools
 ```
 
 <a id="81"></a>
-- Query Device Health Data: **$dmg storage query (device-health | target-health)**
+- Query Device Health Data: **$dmg storage query (device-health**
```

Comment:
nit: Unbalanced parens

Reply:
Thanks Mike, I will correct in the next PR targeted at master.

Reply:
corrected in https://github.com/daos-stack/daos/pull/12079